config/checkstyle/checkstyle.xml

```diff
@@ -29,7 +29,7 @@
 
   <module name="SuppressWarningsFilter" />
   <module name="LineLength">
-    <property name="max" value="130"/>
+    <property name="max" value="160"/>
```

Comment:
can you split the class to another class if have line length more than 130? n revert this pls

Reply:
Refactored to use Functional methods which now works effectively without requiring another class